common/config/rush/pnpm-lock.yaml

```diff
@@ -206,74 +155,72 @@ packages:
-  /@azure/keyvault-keys/4.1.0:
+      integrity: sha512-h58oEROO2tniBTSmFmuHBGvuiFuYsHQBWTVdpT2AiOED4F2Kgf7rs0MPYPXiBcDvihC70M7QPRhIQ3JK1H/ygw==
+  /@azure/logger/1.0.1:
     dependencies:
-      '@azure/core-http': 1.2.1
-      '@azure/core-lro': 1.0.2
-      '@azure/core-paging': 1.1.3
-      '@azure/core-tracing': 1.0.0-preview.9
-      '@azure/logger': 1.0.0
-      '@opentelemetry/api': 0.10.2
-      tslib: 2.0.3
+      tslib: 2.1.0
     dev: false
     engines:
       node: '>=8.0.0'
     resolution:
-      integrity: sha512-xdz7nkYulWSo2+CsBRxaOe2JrGWQP3WxEjPLdIzZNHyaYBuLlnKHL3XWYH5j0UUVoDcmQClZUlZ2P/kdOiy/aQ==
-  /@azure/logger-js/1.3.2:
-    dependencies:
-      tslib: 1.14.1
+      integrity: sha512-QYQeaJ+A5x6aMNu8BG5qdsVBnYBop9UMwgUvGihSjf1PdZZXB+c/oMdM2ajKwzobLBh9e9QuMQkN9iL+IxLBLA==
+  /@azure/ms-rest-azure-env/1.1.2:
     dev: false
     resolution:
-      integrity: sha512-h58oEROO2tniBTSmFmuHBGvuiFuYsHQBWTVdpT2AiOED4F2Kgf7rs0MPYPXiBcDvihC70M7QPRhIQ3JK1H/ygw==
-  /@azure/logger/1.0.0:
+      integrity: sha512-l7z0DPCi2Hp88w12JhDTtx5d0Y3+vhfE7JKJb9O7sEz71Cwp053N8piTtTnnk/tUor9oZHgEKi/p3tQQmLPjvA==
+  /@azure/ms-rest-js/1.9.1_debug@3.2.7:
     dependencies:
+      '@types/tunnel': 0.0.0
+      axios: 0.21.1_debug@3.2.7
+      form-data: 2.5.1
+      tough-cookie: 2.5.0
       tslib: 1.14.1
+      tunnel: 0.0.6
+      uuid: 3.4.0
+      xml2js: 0.4.23
     dev: false
+    peerDependencies:
+      debug: '*'
     resolution:
-      integrity: sha512-g2qLDgvmhyIxR3JVS8N67CyIOeFRKQlX/llxYJQr1OSGQqM3HTpVP8MjmjcEKbL/OIt2N9C9UFaNQuKOw1laOA==
-  /@azure/ms-rest-azure-env/1.1.2:
-    dev: false
-    resolution:
-      integrity: sha512-l7z0DPCi2Hp88w12JhDTtx5d0Y3+vhfE7JKJb9O7sEz71Cwp053N8piTtTnnk/tUor9oZHgEKi/p3tQQmLPjvA==
-  /@azure/ms-rest-js/1.9.0:
+      integrity: sha512-F1crHKhmsvFLM9fsnDyCGFd2E2KR9GEZm5oBVV5D5k2EBQ7u7idtSJlSF6RDLDIrGWtc4NnFdYwsoiW8NLlBQg==
+  /@azure/ms-rest-js/1.9.1_debug@4.3.1:
     dependencies:
       '@types/tunnel': 0.0.0
-      axios: 0.19.2
+      axios: 0.21.1_debug@4.3.1
       form-data: 2.5.1
       tough-cookie: 2.5.0
       tslib: 1.14.1
       tunnel: 0.0.6
       uuid: 3.4.0
       xml2js: 0.4.23
     dev: false
+    peerDependencies:
+      debug: '*'
+    resolution:
+      integrity: sha512-F1crHKhmsvFLM9fsnDyCGFd2E2KR9GEZm5oBVV5D5k2EBQ7u7idtSJlSF6RDLDIrGWtc4NnFdYwsoiW8NLlBQg==
+  /@azure/ms-rest-nodeauth/0.9.3_debug@3.2.7:
+    dependencies:
+      '@azure/ms-rest-azure-env': 1.1.2
+      '@azure/ms-rest-js': 1.9.1_debug@3.2.7
+      adal-node: 0.1.28
+    dev: false
+    peerDependencies:
+      debug: '*'
     resolution:
-      integrity: sha512-cB4Z2Mg7eBmet1rfbf0QSO1XbhfknRW7B+mX3IHJq0KGHaGJvCPoVTgdsJdCkazEMK1jtANFNEDDzSQacxyzbA==
-  /@azure/ms-rest-nodeauth/0.9.3:
+      integrity: sha512-aFHRw/IHhg3I9ZJW+Va4L+sCirFHMVIu6B7lFdL5mGLfG3xC5vDIdd957LRXFgy2OiKFRUC0QaKknd0YCsQIqA==
+  /@azure/ms-rest-nodeauth/0.9.3_debug@4.3.1:
```

Comment:
Does this version make sense? cc @ramya-rao-a @xirzec

Reply:
What does the `_debug` suffix even mean?